src/NuGet.Clients/NuGet.CommandLine/Commands/Command.cs

```diff
@@ -261,5 +281,17 @@ public virtual CommandAttribute GetCommandAttribute()
             }
             return null;
         }
+
+        public DeprecatedCommandAttribute GetDeprecatedCommandAttribute()
```

Comment:
Either this shouldn't be public or delete the property that calls it.

Reply:
I was inspired by the previous method [`GetCommandAttribute()`](https://github.com/NuGet/NuGet.Client/blob/132a4493d44ba9c4ae2e7cfe5a9efd1538c7fc1a/src/NuGet.Clients/NuGet.CommandLine/Commands/Command.cs#L243)

Reply:
I think it's bad there too :)

Reply:
Ok. I will make the method private, since there's no usage outside this class.